src/rendering/RenderUtil.cc

```diff
@@ -680,9 +686,7 @@ void RenderUtil::Update()
     if (scene.Grid() && !this->dataPtr->enableSensors)
       this->ShowGrid();
     if (scene.Sky())
-    {
```

Comment:
Restore

Reply:
I got codecheck error that RenderUtil::Update function has exceed 500 line, so that is why i removed these and the others
is that ok ?